packages/api/tests/lambdas/test-bulk-operation.js

```diff
@@ -710,11 +710,16 @@ test.serial('bulk operation BULK_GRANULE_REINGEST reingests granules returned by
   t.true(esSearchStub.called);
   t.is(reingestStub.callCount, 2);
 
-  reingestStub.args.forEach((callArgs) => {
+  reingestStub.args.forEach(async (callArgs) => {
```

Comment:
This....matches master, but shouldn't these promises be awaited?

Reply:
So I'm thinking we're doing it this way because we need to get the `pgGranule` and then translate it synchronously, then check the result also synchronously. If we did a Promise.all we could guarantee order. 

Is that what you're saying? The callback function here is `async` but not returning anything so I don't think we need to await? It may still return a promise but not sure it matters.